### ISSUE TITLE
investigate slow query

### ISSUE THREAD

**Issue by missinglink** [pelias/placeholder#209]
```
placeholder > 3033 Cleveland Avenue Redwood Credit Union, Santa Rosa, CA 95403
 - 85922693	locality 	Santa Rosa
 - 101724225	locality 	Santa Rosa
 - 421189199	localadmin 	Santa Rosa
 - 421200019	locality 	Santa Rosa de Calamuchita
 - 421200057	locality 	Santa Rosa
 - 1092076157	county 	Santa Rosa
 - 1108700489	county 	Piamonte
 - 1108700503	county 	Santa Rosa
 - 1126089665	neighbourhood 	Santa Rosa
 - 1193122923	locality 	Santa Rosa
 - 1209216983	locality 	Santa Rosa
 - 1225816779	locality 	Santa Rosa Numero Uno
 - 1225849937	locality 	Santa Rosa
 - 1226782595	locality 	Santa Rosa
 - 1243157887	locality 	Santa Rosa
 - 1243257999	locality 	Santa Rosa Baja
 - 1259572505	locality 	Santa Rosa
 - 1259686277	locality 	Santa Rosa
 - 1259696555	locality 	Santa Rosa
 - 1259777393	locality 	Santa Rosa
 - 1276633819	locality 	Santa Rosa
 - 1276769087	locality 	Santa Rosa
 - 1276989785	locality 	Santa Rosa
 - 1293200495	locality 	Santa Rosa
 - 1293678485	locality 	Santa Rosa
 - 1293929445	locality 	Santa Rosa
 - 1310693239	locality 	Santa Rosa
 - 1326724733	locality 	Santa Rosa
 - 1327317847	locality 	Santa Rosa
 - 1327341399	locality 	Santa Rosa
 - 1343392715	locality 	Santa Rosa
 - 1343450355	locality 	Santa Rosa
 - 1343482881	locality 	Santa Rosa
 - 1343681981	locality 	Santa Rosa
 - 1343753617	locality 	Santa Rosa
 - 1344154503	locality 	Santa Rosa
 - 1344165857	locality 	Santa Rosa
took: 9.019s
```

**Comment by missinglink** [comment]
proposed solution: https://github.com/pelias/placeholder/pull/210

```bash
npm run cli -- 3033 Cleveland Avenue Redwood Credit Union, Santa Rosa, CA 95403

> pelias-placeholder@0.0.0-development cli
> node cmd/cli.js "3033" "Cleveland" "Avenue" "Redwood" "Credit" "Union," "Santa" "Rosa," "CA" "95403"

3033 Cleveland Avenue Redwood Credit Union, Santa Rosa, CA 95403

took: 330.55ms
 - 85922693	locality 	Santa Rosa
 - 101724225	locality 	Santa Rosa
 - 421189199	localadmin 	Santa Rosa
 - 421200019	locality 	Santa Rosa de Calamuchita
 - 421200057	locality 	Santa Rosa
 - 1092076157	county 	Santa Rosa
 - 1108700489	county 	Piamonte
 - 1108700503	county 	Santa Rosa
 - 1126089665	neighbourhood 	Santa Rosa
 - 1193122923	locality 	Santa Rosa
 - 1209216983	locality 	Santa Rosa
 - 1225816779	locality 	Santa Rosa Numero Uno
 - 1225849937	locality 	Santa Rosa
 - 1226782595	locality 	Santa Rosa
 - 1243157887	locality 	Santa Rosa
 - 1243257999	locality 	Santa Rosa Baja
 - 1259572505	locality 	Santa Rosa
 - 1259686277	locality 	Santa Rosa
 - 1259696555	locality 	Santa Rosa
 - 1259777393	locality 	Santa Rosa
 - 1276633819	locality 	Santa Rosa
 - 1276769087	locality 	Santa Rosa
 - 1276989785	locality 	Santa Rosa
 - 1293200495	locality 	Santa Rosa
 - 1293678485	locality 	Santa Rosa
 - 1293929445	locality 	Santa Rosa
 - 1310693239	locality 	Santa Rosa
 - 1326724733	locality 	Santa Rosa
 - 1327317847	locality 	Santa Rosa
 - 1327341399	locality 	Santa Rosa
 - 1343392715	locality 	Santa Rosa
 - 1343450355	locality 	Santa Rosa
 - 1343482881	locality 	Santa Rosa
 - 1343681981	locality 	Santa Rosa
 - 1343753617	locality 	Santa Rosa
 - 1344154503	locality 	Santa Rosa
 - 1344165857	locality 	Santa Rosa
```

**Comment by orangejulius** [comment]
I think we solved this one with #210
